client/signup/steps/jpo-connect/style.scss

```diff
@@ -0,0 +1,36 @@
+.jpo__summary-wrapper {
```

Comment:
We should be styling these components on their own stylesheets. These seem to only target elements in other components. I recommend moving all of them.